packages/oauth/src/stores/file-store.ts

```diff
@@ -105,36 +105,28 @@ export default class FileInstallationStore implements InstallationStore {
       filesToDelete = filesToDelete.concat(userFiles);
     }
 
-    try {
-      filesToDelete.map((filePath) => deleteFile(path.resolve(`${installationDir}/${filePath}`)));
-    } catch (err) {
-      throw new Error('Failed to delete installation from FileInstallationStore');
-    }
+    filesToDelete.forEach((filePath) => {
```

Comment:
The error handling here should be done for each deletion

Reply:
Doesn't this technically change the behaviour compared to the previous version? In the previous version, if `path.resolve` or `fs.unlink` would error, an exception was raised, whereas in this PR's version, no exception is thrown and an error is logged out instead.

Reply:
@filmaj Good point. I can revert the logic change here and add only log improvement this time.

Reply:
This has been improved